apps/admin_app/assets/js/views/shipping_policy/edit.js

```diff
@@ -0,0 +1,69 @@
+import MainView from '../main';
+
+export default class View extends MainView {
```

Comment:
@anantanant2015 can you please make this big piece of a code a function and call it in the two files. This will remove code repetition.